docs/kbs.yaml

```diff
@@ -259,6 +270,15 @@ components:
           description: >-
             Encryption algorithm used to encrypt the response payload.
 
+    AttestationPolicy:
+      required:
+        - policy
+      properties:
+        policy:
+          type: string
+          description: >-
+            Base64 encoded attestation verification policy.
```

Comment:
Which policy format would you recommend? OPA? We should document which ones KBS supports.

Reply:
I have changed the input here now to specify the type of policy engine and the id of the policy :)